tests/python/PyreBearTest.py

```diff
@@ -0,0 +1,45 @@
+import os
+import unittest
+from queue import Queue
+
+from bears.python.PyreBear import PyreBear
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.settings.Section import Section
+
+
+def get_absolute_test_path(file):
+    return os.path.join(os.path.dirname(__file__),
+                        'pyre_test_files', file)
+
+
+@generate_skip_decorator(PyreBear)
+class PyreBearTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section('pyre')
+        self.file_dict = {}
+        self.queue = Queue()
+        test_folder = os.path.join(
+            os.path.dirname(__file__), 'pyre_test_files')
+        self.test_file = os.path.join(test_folder, 'pyre_test.py')
+
+    def get_results(self, file):
+        with open(file, 'r', encoding='utf-8') as content:
+            self.file_dict[file] = tuple(content.readlines())
+
+        self.uut = PyreBear(self.file_dict,
+                            self.section,
+                            self.queue)
+
+        return list(self.uut.run_bear_from_section([], {}))
+
+    def test_results(self):
+        # Get result from bear
+        results = self.get_results(self.test_file)
+        # Check result is as expected
+        messages = [result.message for result in results]
+        self.assertEqual(len(messages), 2)
+        self.assertEquals(messages[0], 'Incompatible return type [7]: '
+                          'Expected `int` but got `str`.')
+        self.assertEquals(messages[1], 'Incompatible return type [7]: '
```

Comment:
Apart from the message, we'd better also verify line & column number. This will be useful for reviewers to check if the linter is using 1-based line & column number convention. Also, if we bump to a new version where they change their line or column number convention, CI would fail to remind us.